PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -178,11 +179,6 @@
   jercCHF   = jetTable.variables.jercCHF,
 )
 
-for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016:
```

Comment:
You are removing this also for "run2_miniAOD_80XLegacy": is it wanted? And why, if so?

Reply:
@perrotta Ah its a mistake on my part to remove it. I thought ``` run_jme_2016``` is supposed to also replace ```run2_miniAOD_80XLegacy```. Can you confirm that this is not the case? I can put it back in.

Reply:
It could be... @srimanob, could you please confirm if the ```run2_jme_2016``` modifier is supposed to replace ```run2_miniAOD_80XLegacy```? Then I assume that ```run2_nanoAOD_94X2016``` will be replaced by ```run2_jme_2017```, correct?

In any case, wouldn't it be wise to maintain in these scripts also the previous customizations for ```run2_miniAOD_80XLegacy``` and  ```run2_jme_2016```, since those two modifiers still exist and can be used?

Reply:
I followed the same changes in the latest releases as can be seen [here](https://github.com/cms-sw/cmssw/blob/CMSSW_11_1_0_pre2/PhysicsTools/NanoAOD/python/jets_cff.py#L75-L78) in CMSSW_11_1_0_pre2, which you can compare to [here](https://github.com/cms-sw/cmssw/blob/CMSSW_10_2_18/PhysicsTools/NanoAOD/python/jets_cff.py#L76-L81) in CMSSW_10_2_18. Maybe @peruzzim can comment on this.

Reply:
> I followed the same changes in the latest releases as can be seen [here](https://github.com/cms-sw/cmssw/blob/CMSSW_11_1_0_pre2/PhysicsTools/NanoAOD/python/jets_cff.py#L75-L78) in CMSSW_11_1_0_pre2, which you can compare to [here](https://github.com/cms-sw/cmssw/blob/CMSSW_10_2_18/PhysicsTools/NanoAOD/python/jets_cff.py#L76-L81) in CMSSW_10_2_18. Maybe @peruzzim can comment on this.

The changes as proposed in this PR, as well as those already implemented in the link provided by @nurfikri89 , imply that the eras ```run2_miniAOD_80XLegacy``` and ```run2_nanoAOD_94X2016``` are not going to be used any more in any central or even user workflow meant to re-miniAOD or re-nanoAOD of 2016-2017 data.

If this is the case, I am perfectly fine with the current implementation. However, if so, it would probably be wise to clean up the release everywhere for those unused eras (there are several python configs which still use it) 

Please @peruzzim comment on this point.

Reply:
@perrotta we have introduced a few months ago the following splitting of modifiers:

- `run2_miniAOD_80XLegacy`, `run2_nanoAOD_94X2016` and similar: these identify the version of the MiniAOD input file;
- `run2_jme_2016`, `run2_jme_2017` and similar: these identify the data taking year.

The jet ID requirements depend on the year, not on the MiniAOD version (for instance, we want to run the same set of cuts for 2017, be it in 10_2_X or in UL). The former modifiers are still needed somewhere else in the code to trigger recalculation of quantities that were not stored in MiniAOD at the time that was run, or need updating for some reason.

(sorry for the delay in giving this clarification)

Reply:
@perrotta @peruzzim @nurfikri89 

> The former modifiers are still needed somewhere else in the code to trigger recalculation of quantities that were not stored in MiniAOD at the time that was run, or need updating for some reason.

Does this mean we need to update this PR to include this case. ie. run2_miniAOD_80XLegacy triggering run2_jme_2016 and run2_nanoAOD_94X2016 triggering run2_jme_2017 ?

Reply:
for jet IDs we should just use run2_jme_2016 and run2_jme_2017 because they depend only on the year. we should not introduce any chaining of modifiers.

Reply:
Thinking towards the near future. How will we handle the fact that EOY jet ID criteria will be different from UL jet ID criteria?